.github/markdown_lint_config.json

```diff
@@ -29,7 +29,7 @@
   "MD028": false,
   "MD029": false,
   "MD030": false,
-  "MD031": true,
+  "MD031": false,
```

Comment:
can we ignore this rule for certain folder?

Reply:
Any suggestions for how to define the ignore specific folder?

Reply:
will https://github.com/igorshubovych/markdownlint-cli?tab=readme-ov-file#ignoring-files this help?